test/openAPI/features/wallet_generate_link_code.feature

```diff
@@ -0,0 +1,24 @@
+@method=POST @endpoint=/linked-authorization/link-code
+Feature: The endpoint to generate link code.
+
+  @smoke @unit @positive
+  Scenario: Successfully generate link code smoke type test
+    Given Wants to generate link code
+    When Send POST /linked-authorization/link-code request with given X-XSRF-TOKEN header, transactionId and requestTime
+    Then Receive a response from the /linked-authorization/link-code endpoint
+    And The /linked-authorization/link-code endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-code endpoint response should have status 200
+    And The /linked-authorization/link-code endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-code endpoint response should match json schema
+    And The /linked-authorization/link-code response should contain transactionId property equals provided transactionId
+
+  @negative
+  Scenario: Not able to generate link code because of invalid transaction_id
+    Given Wants to generate link code
+    When Send POST /linked-authorization/link-code request with given X-XSRF-TOKEN header, invalid transactionId and requestTime
+    Then Receive a response from the /linked-authorization/link-code endpoint
+    And The /linked-authorization/link-code endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-code endpoint response should have status 200
+    And The /linked-authorization/link-code endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-code endpoint response should match json schema
+    And The /linked-authorization/link-code response should contain errorCode property equals to "invalid_transaction_id"
```

Comment:
Can we also have separate test cases for invalid request time and invalid xsrf token id ?

Reply:
Of course, what error code to give for invalid request time and invalid xsrf token id?
In the spec we have now [invalid_transaction_id, invalid_link_code, response_timeout, unknown_error]

Reply:
invalid request time - we can give invalid_request error code, we have this in the latest spec.
invalid xsrf token - we can assert for a 403 http response code

Reply:
Changes are available in https://github.com/GovStackWorkingGroup/bb-identity/pull/42